fe/fe-core/src/test/java/com/starrocks/sql/analyzer/AnalyzeUpdateTest.java

```diff
@@ -0,0 +1,47 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.sql.analyzer;
+
+import com.starrocks.utframe.UtFrameUtils;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.File;
+import java.util.UUID;
+
+import static com.starrocks.sql.analyzer.AnalyzeTestUtil.analyzeFail;
+import static com.starrocks.sql.analyzer.AnalyzeTestUtil.analyzeSuccess;
+
+public class AnalyzeUpdateTest {
+    // may also start a Mocked Frontend
+    private static String runningDir = "fe/mocked/AnalyzeUpdate/" + UUID.randomUUID().toString() + "/";
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        UtFrameUtils.createMinStarRocksCluster(runningDir);
+        AnalyzeTestUtil.init();
+    }
+
+    @AfterClass
+    public static void tearDown() {
+        File file = new File(runningDir);
+        file.delete();
+    }
+
+    @Test
+    public void testSingle() {
+        analyzeFail("update tjson set v_json = '' where v_int = 1",
```

Comment:
How about where v_int in (1, 2, 3)

Reply:
this test is used to verify update syntax only support the primary key table, so the kind of predicate is irrelevant.